docs/faq/README.md

```diff
@@ -87,6 +64,42 @@ Regardless of storage option used, IPAM controller expects a directory volume mo
 
 In the mount directory, rename or remove a file named `cis_ipam.sqlite3`.
 
+## Troubleshooting
+
+### How to troubleshoot FIC pod logs ?
+
+`kubectl logs deploy/<name-of-ipam-deployment> -n kube-system -f`
+
+### Error - `Unable to Update IPAM: kube-system/***  Error: ipams.fic.f5.com "***" not found`
+
+* Verify cluster role permissions for ipam resource.
+
+  - `kubectl describe clusterrole <name> -n kube-system | grep ipam`
+
+* Delete existing IPAM CRD and re-start both CIS and IPAM pods
+  * Scale down CIS controller Deployment
+    * `kubectl scale deploy/bigip-controller -n kube-system --replicas=0`
+  * Scale down FIC controller Deployment
+    * `kubectl scale deploy/ipam-controller -n kube-system --replicas=0`
+  * `kubectl delete crd ipams.fic.f5.com`
+  * If you are upgrading from CIS 2.5.0, FIC 0.1.4, then make sure f5ipams CRD is unlinked completely.
+    * `kubectl delete crd f5ipams.fic.f5.com`
+  * Scale up CIS controller Deployment
+    * `kubectl scale deploy/bigip-controller -n kube-system --replicas=1`
```

Comment:
May be not always the deployment name is bigip-controller. Please share its an examples and replace `deploy/<your-cis-deploy-name>`